### PR TITLE
Add init container condition

### PR DESCRIPTION
**Pull request by rmn-boiko**

### CODE REVIEW

File: charts/kamu-api-server/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kamu-api-server
 description: API server component of the Kamu Compute Node
 type: application
-version: 0.28.2
+version: 0.28.3
 appVersion: "0.28.2"
```
Comment (s373r) [inline charts/kamu-api-server/Chart.yaml:6]: please update `appVersion` as well

File: charts/kamu-api-server/values.yaml [hunk 2]
```diff
@@ -49,6 +49,7 @@ imageMigrations:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+enableDatabase: false
```
Comment (s373r) [inline charts/kamu-api-server/values.yaml:52]: **Q:** why do we need this switch? we want to do a [future] deployment on `demo` without a DB?

File: charts/kamu-api-server/Chart.yaml (hunk 1 above)
Comment (s373r) [reply charts/kamu-api-server/Chart.yaml:6]: the chart changes (the template in our case), so the version does too

**Review by s373r — CHANGES_REQUESTED** [review]
> PR name: Upgrade demo env version

Please note: `helm-charts` repo is environment-independent

File: charts/kamu-api-server/values.yaml (hunk 2 above)
Comment (rmn-boiko) [reply charts/kamu-api-server/values.yaml:52]: Yes. exactly. Currently only europort env will be with database

File: charts/kamu-api-server/Chart.yaml (hunk 1 above)
Comment (s373r) [reply charts/kamu-api-server/Chart.yaml:6]: I mixed up `version` & `appVersion`: the comment is not actual anymore

**Review by s373r — APPROVED** [review]

**Pull request merged**